app/src/main/res/layout/activity_opportunities_activity.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".opportunities_activity">
+
+
+    <Button
+        android:id="@+id/button2"
```

Comment:
not sure what a better id for this button is but can we use something other than `button2`?